tasks/libs/common/omnibus.py

```diff
@@ -113,6 +113,7 @@ def env_filter(item):
             "ENVIRONMENTS",
             "EXPERIMENTS_EVALUATION_ADDRESS",
             "FILTER",
+            "FIPS_MODE",
```

Comment:
Since there's an extra software being built when FIPS is enabled, I think we need to account for it when computing the cache key (ie. you should remove this line from the ignore list)

Reply:
removed in https://github.com/DataDog/datadog-agent/pull/31004/commits/9b56d4ede78dac2a6e16330fc01c157c61064699